### PR TITLE
docs(RELEASE_TEMPLATE): clarify publishing steps

### PR DESCRIPTION
**Pull request by nikku**
This improves the current publishing steps:

* updates to latest blog publishing flow
* simplifies getting started with the blog by using a tailored template

### CODE REVIEW

File: docs/.project/RELEASE_TEMPLATE.md [hunk 1]
```diff
@@ -57,18 +57,16 @@ _To be done once the release is built._
 
 * [ ] publish release on [Github Releases](https://github.com/camunda/camunda-modeler/releases)
 * [ ] trigger mirroring of release to [Camunda Download Center](https://downloads.camunda.cloud/release/camunda-modeler/) via [Jenkins](https://ci.cambpm.camunda.cloud/job/sideprojects/job/camunda-modeler-desktop-RELEASE/build?delay=0sec)
-* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/), [see guidelines](https://confluence.camunda.com/display/HAN/Your+Content+on+the+Camunda+Blog)
-   * [ ] use the [blog post template](https://docs.google.com/document/d/18aXWFBdbET97EZ_HO1DEVLVsYP-UQEabumSzQdeRbDw/edit?usp=sharing)
-   * [ ] add your contents to the newly created post
-   * [ ] set yourself as the post author
+* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/) according to [guidelines](https://confluence.camunda.com/display/HAN/Submitting+a+blog+post):
```
Comment (barmac) [inline docs/.project/RELEASE_TEMPLATE.md:60]: Use permalink:

```suggestion
* [ ] write blog post for the [Camunda Blog](https://camunda.com/blog/) according to [guidelines](https://confluence.camunda.com/x/Wi_SBg):
```

File: docs/.project/RELEASE_TEMPLATE.md [hunk 2]
```diff
@@ -57,18 +57,16 @@ _To be done once the release is built._
...
    * [ ] [camunda-docs-static](https://github.com/camunda/camunda-docs-static)
    * [ ] [camunda-docs-manual](https://github.com/camunda/camunda-docs-manual)
    * [ ] [camunda-docs-platform](https://github.com/camunda/camunda-platform-docs)
-* [ ] send all information required to compile the [release notice email](https://github.com/bpmn-io/internal-docs/blob/master/releases/modeler/CAMUNDA_MODELER.md#release-notice-email) to the [`@product-release-presentation-dri` role](https://confluence.camunda.com/pages/viewpage.action?spaceKey=HAN&title=Release+Presentation+Process#ReleasePresentationProcess-OrganisingtheReleasePresentation)
-* [ ] request marketing to update the [downloads page](https://camunda.com/download/modeler/) via [request form](https://confluence.camunda.com/display/MAR/Marketing+Request+Form)
+* [ ] provide content to the [release presentation and notice](https://confluence.camunda.com/pages/viewpage.action?spaceKey=HAN&title=Release+Presentation+Process#ReleasePresentationProcess-OrganisingtheReleasePresentation)
```
Comment (barmac) [inline docs/.project/RELEASE_TEMPLATE.md:68]: Use permalink:

```suggestion
* [ ] provide content to the [release presentation and notice](https://confluence.camunda.com/x/Uq-gBQ#ReleasePresentationProcess-OrganisingtheReleasePresentation)
```

File: docs/.project/RELEASE_TEMPLATE.md [hunk 3]
```diff
@@ -57,18 +57,16 @@ _To be done once the release is built._
...
+* [ ] trigger [downloads page](https://camunda.com/download/modeler/) update via [marketing request form](https://confluence.camunda.com/display/MAR/Marketing+Request+Form)
```
Comment (barmac) [inline docs/.project/RELEASE_TEMPLATE.md:69]: Use permalink

```suggestion
* [ ] trigger [downloads page](https://camunda.com/download/modeler/) update via [marketing request form](https://confluence.camunda.com/x/rTGSBg)
```

**Comment by nikku** [comment]
Squashed the changes via https://github.com/camunda/camunda-modeler/pull/3454/commits/026a7d79090035a88c11b1d173e7adb7a836e6b8.

**Review by barmac — APPROVED** [review]

**Pull request merged**